bool cutter_left
bool cutter_right
